README.md

```diff
@@ -0,0 +1,37 @@
+# Openconfig Security Services
+
+## SBOM
+
+### CLI
+
+#### Overview
+
```

Comment:
Is the intent that OpenConfig adopts the [SPDX](https://spdx.dev/) format for SBOM?

Reply:
both cyclone dx and spdx are valid formats from the ntia